data/README.md

```
@@ -0,0 +1,161 @@
+## Інструменти
+
+### `tools/convert_region_map.py`
+Python скрипт, що конвертує JSON дані у формат C++ заголовку.
+
+**Використання:**
+```bash
+python3 tools/convert_region_map.py [input.json] [output.h]
+```
+
+**Вивід скрипта:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)